### PR TITLE
Add documentation for Smart Meter Texas integration

### PR DESCRIPTION
**Pull request by grahamwetzler**
## Proposed change

Add documentation for the Smart Meter Texas integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/37966
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1780
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/smart_meter_texas.markdown [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+<div class='note'>
+Due to API limits, this sensor refreshes once per hour. The limit is 24 meter readings per calendar day.
+</div>
+
+## Configuration
+
+This integration can be configured from the frontend.
+
+Menu: _Configuration > Integrations_
```
Comment (klaasnicolaas) [inline source/_integrations/smart_meter_texas.markdown]: And click on the `+` sign and search for `Smart Meter Texas`...?

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/smart_meter_texas.markdown (hunk 1 above)
Comment (grahamwetzler) [reply source/_integrations/smart_meter_texas.markdown]: Sorry about that. Hopefully it looks good now?

File: source/_integrations/smart_meter_texas.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ha_codeowners:
+  - "@grahamwetzler"
+ha_domain: smart_meter_texas
+---
+
+The [Smart Meter Texas](https://www.smartmetertexas.com/) integration allows Texas residents to integrate their electric meters into Home Assistant. This integration will create a sensor for each meter that shows the current reading.
+
+On its own this sensor is not particularly useful. However, combined with a [`utility_meter`](https://www.home-assistant.io/integrations/utility_meter/) sensor you can obtain usage over a month, for example.
+
+Any Texas electricity consumer can [register an account](https://www.smartmetertexas.com/register). To register you will need following details which can be found on your electric bill.
```
Comment (mrand) [inline source/_integrations/smart_meter_texas.markdown]: https://www.smartmetertexas.com/register seems to redirect to /home and might change in the future.  Perhaps just go without the extended path?

Comment (grahamwetzler) [reply source/_integrations/smart_meter_texas.markdown]: Good catch. I removed the /register link and provided better instructions to setup an account.

**Review by fabaff — APPROVED** [review]

**Pull request merged**